biodatacatalyst-ui/src/main/webapp/picsureui/search-interface/filter-model.js

```diff
@@ -202,7 +202,7 @@ define(["backbone", "handlebars", "picSure/settings", "picSure/queryBuilder", "o
             generateDatatableCategory: function(searchResult) {
                 return "\\" + searchResult.result.dtId + "\\" + searchResult.result.studyId + "\\";
             },
-            addExportColumn: function(searchResult, type, source){
+            addExportColumn: function(searchResult, type, source, selectedValues, selectedMin, selectedMax){
                 let existingColumn =   _.find(this.get('exportColumns').models, (model)=>{
                          return model.attributes.variable.metadata.columnmeta_var_id === searchResult.result.metadata.columnmeta_var_id;
                 });
```

Comment:
You could create an object for the exportColumns up here, since you'll likely use it. Then use it like`this.get('exportColumns).add(exportColumns);`

```
const exportColumns = {
  type: type,
  variable: searchResult.result,
  source: source,
  selectedValues: selectedValues,
  selectedMin: selectedMin,
  selectedMax: selectedMax
};
```